src/site/content/en/blog/async-clipboard/index.md

```diff
@@ -0,0 +1,413 @@
+the promise to reject so the page can respond appropriately.
+
+Because Chrome only allows clipboard access when a page is the active tab,
+you'll find that some of the examples here don't run if pasted directly into
+DevTools, since DevTools itself is the active tab. There's a trick: defer
+clipboard access using `setTimeout()`, then quickly click inside the page to
+focus it before the functions are called:
+
+```js
+setTimeout(async () => {
+  const text = await navigator.clipboard.readText();
+  console.log(text);
+}, 2000);
+```
+
+## Feature detection
+
+To use the Async Clipboard API while supporting all browsers, test for
+`navigator.clipboard` and fall back to earlier methods. For example, here's how
+you might implement pasting to include other browsers.
+
+```js
+document.addEventListener('paste', async e => {
+  e.preventDefault();
+  let text;
+  if (navigator.clipboard) {
+    text = await navigator.clipboard.readText()
+  }
+  else {
+    text = e.clipboardData.getData('text/plain');
+  }
+  console.log('Got pasted text: ', text);
+});
+```
+
+That's not the whole story. Before the Async Clipboard API, there were a mix of
+different copy and paste implementations across web browsers. In most browsers,
+the browser's own copy and paste can be triggered using
+d`ocument.execCommand('copy')` and `document.execCommand('paste')`. If the text
+to be copied is a string not present in the DOM, it must be injected into the
+DOM and selected:
+
+```js
+button.addEventListener('click', e => {
+  const input = document.createElement('input');
+  document.body.appendChild(input);
+  input.value = text;
+  input.focus();
+  input.select();
+  const result = document.execCommand('copy');
+  if (result === 'unsuccessful') {
+    console.error('Failed to copy text.');
+  }
+});
+```
+
+In Internet Explorer, I can also access the clipboard through
+`window.clipboardData`. If accessed within a user gesture such as a click
+event—part of asking permission responsibly—no permissions prompt is shown.
+
+## Demos
+
+You can play with the Async Clipboard API in the demos below. You might get a
+`NotAllowedError` if running in an iframe, that is caused the newly implemented
+[clipboard Feature
+Policy](https://www.chromestatus.com/feature/5767075295395840). In this case
+just run the demo [directly on Glitch](https://async-clipboard-api.glitch.me/).
+
+The first example demonstrates moving text on and off the clipboard.
+
+<div class="glitch-embed-wrap" style="height: 500px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/async-clipboard-text?previewSize=100"
+    title="async-clipboard-text on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media;
+clipboard; clipboard-read; clipboard-write"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+To try the API with images use this demo. Recall that only pngs are supported
```

Comment:
```suggestion
To try the API with images use this demo. Recall that only PNGs are supported
```

Reply:
The PNG limitation seems in contradiction with what you wrote above: "Chrome 84 introduces an experimental feature that allows the clipboard to handle any arbitrary data type".